ccmlib/cluster.py

```diff
@@ -720,3 +720,11 @@ def error(self, message):
     @staticmethod
     def is_docker():
         return False
+
+    @property
+    def tablets_enabled(self) -> bool:
+        return 'tablets' in self._config_options.get('experimental_features', [])
+
+    @property
```

Comment:
I think we should do the c-s default, which is 1

Reply:
true. Fixed.